docs/opentitan/flashgen.md

```diff
@@ -6,33 +6,41 @@ flash controller virtual device.
 ## Usage
 
 ````text
-usage: flashgen.py [-h] [-D] [-x file] [-l file] [-b {0,1}] [-s OFFSET] [-v] [-d] flash
+usage: flashgen.py [-h] [-D] [-a {0,1}] [-s OFFSET] [-f FLASHBASE] [-x file] [-X elf] [-b file] [-B elf] [-v] [-d] [-U] flash
```

Comment:
I don't see `-f FLASHBASE` anywhere in the code, is it implemented?

Reply:
Oops, I forgot to update the documentation. Good catch.